content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -137,7 +137,7 @@ collection, which deletes images in order based on the last time they were used,
 starting with the oldest first. The kubelet deletes images
 until disk usage reaches the `LowThresholdPercent` value.
 
-### Container image garbage collection {#container-image-garbage-collection}
+### Container garbage collection {#container-image-garbage-collection}
```

Comment:
I suggest we remove the custom fragment identifier `#container-image-garbage-collection` too.